examples/multiple_tables/README.md

```diff
@@ -8,6 +8,7 @@ This example is a good reference to understand and test the module usage.
 
 | Name | Description | Type | Default | Required |
 |------|-------------|:----:|:-----:|:-----:|
+| authorized\_views | An object defining an authorized view to create. | object | `<list>` | no |
```

Comment:
Instead of putting the entire config into a variable, let's hard-code a sample view into the example `main.tf`. (I know this isn't the existing pattern, but the existing pattern is an anti-pattern.)

Reply:
Gotcha, done.